.buildconfig

```diff
@@ -0,0 +1,10 @@
+[default]
+name=Default
+runtime=host
+config-opts=
+run-opts=
+prefix=/home/elza/.cache/gnome-builder/install/plume/host
```

Comment:
I don't know what buildconfig is for, but it seems quiet tied to your specific configuration, can't it be made otherwise?